airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -41,9 +42,9 @@ interface FormikConnectionFormValues {
   normalization?: NormalizationType;
 }
 
-type ConnectionFormValues = ValuesProps;
+export type ConnectionFormValues = ValuesProps;
```

Comment:
Looking through the code, I _believe_ what happens is the `FormikConnectionFormValues` are what we use to interact with the form, then there's some casting/transforming and `ConnectionFormValues` is what is submitted to the API?  

I think we should add a comment explaining that in here (or if I'm wrong, add a comment explaining the right thing :) )

Reply:
https://github.com/airbytehq/airbyte/blob/923c7cbec3193605c8c4668291403a2c2df90f5a/airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx#L172
This?

Reply:
Yeah.  Just thinking if it took me a few minutes to figure that out, we could save future someones a few minutes each by commenting why there are two types for values.

Reply:
I have a TODO comment in the main PR about aligning the types. Might not be easy though.

Reply:
Gotcha.  We can resolve this then since it sounds like there's some clarification in the other PR.  Thanks!